src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -6,6 +6,8 @@
 
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Versioning;
+using System.Threading;
```

Comment:
You shouldn't need new `using`s here. Refs always use fully qualified names (e.g. `System.IO.Stream` instead of just `Stream`). You are supposed to actually generate them instead of changing by hand: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md#for-most-assemblies-within-libraries